src/solver/constraints-builder/cbuilder.h

```diff
@@ -291,7 +291,7 @@ class CBuilder final
 
     linkInfo* getLink(int i)
     {
-        if (i < pLink.size())
+        if (i < (int)pLink.size())
             return pLink[i];
         return nullptr;
     }
```

Comment:
I suggest passing an `uint` to the `getLink` function instead

Reply:
Done.